### DISCUSSION TITLE
Table doesn't show in hover editor

### DISCUSSION THREAD

**Discussion by dcoales** [RafaelGB/obsidian-db-folder#202] [Q&A]
If I hover over a link to a database file then the hover editor appears but is empty.  To see the grid I have to switch to markdown mode and then back to database view at which point the table renders correctly.

This happens to me on a windows desktop running version 0.15.4 of Obsidian, version 1.8.2 of DB Folder and version 0.10.7 of Hover Editor.

Awesome plugin BTW !

**Comment by RafaelGB** [comment]
The hover link of the database just render a dataview table (read only)
https://github.com/RafaelGB/obsidian-db-folder/issues/41

not sure if hover edition supports wraps on the same hover action ( same strategy of excalidraw plugin)

**Comment by dcoales** [comment]
Hi Rafael,

I'm not sure if i explained the issue properly.  It's not that the database doesn't display well in the hover editor plugin - it works fine.  It's just that it doesn't render when the window first opens.  You have to toggle to markdown view and then back to database view in the hover editor before it will render.  Once rendered it's fine.

Cheers,
David

**Reply by RafaelGB** [reply]
Hmm, I will try to test it as soon as I can.

To facilitate the process, could you share here your ddbb file?
